src/test/java/database/helper/SettingsTest.java

```diff
@@ -3,6 +3,7 @@
 import org.junit.jupiter.api.Test;
 
 import static database.helper.Settings.DEFAULT_DATABASE_STORAGE_PATH;
+import static database.helper.Settings.DEFAULT_LIMIT_VALUE;
 import static org.junit.jupiter.api.Assertions.assertEquals;
 
 public class SettingsTest {
```

Comment:
FYI: you don't have to make test classes `public`. You can use default modificator as for test methods.

Reply:
Discussed offline. Please make this test class and all others non-public.